src/sidebar.js

```diff
@@ -312,7 +311,8 @@ class Sidebar extends React.Component {
 
     return (
       <div {...rootProps}>
-        <div className={this.props.sidebarClassName} style={sidebarStyle} ref="sidebar">
+        <div className={this.props.sidebarClassName} style={sidebarStyle}
+             ref={node => (this.sidebar = node)}>
```

Comment:
We should cache this function on the instance to avoid recreating this method on each invocation of `render`